sycamore/docset.py

```diff
@@ -17,7 +17,10 @@
 
 
 class DocSet:
-    """DocFrame is a distributed computation framework for Documents."""
+    """
+    A DocSet, short for “documentation set,” is a distributed collection of documents bundled together for processing.
```

Comment:
Doesn't have to come right away, but I think eventually we may want method documentation for DocSet. To some extent that may be more useful than the transform docs, since most users will probably invoke the transforms via the DocSet APIs.

Reply:
Updated most of them